src/puppetlabs/puppetdb/config.clj

```diff
@@ -133,13 +144,17 @@
      :threads (pls/defaulted-maybe s/Int half-the-cores)
      :store-usage s/Int
      :max-frame-size (pls/defaulted-maybe s/Int 209715200)
-     :temp-usage s/Int}))
+     :temp-usage s/Int
+     :max-command-size (pls/defaulted-maybe s/Num (default-max-command-size))
+     :enable-max-command-size-fail (pls/defaulted-maybe s/Bool false)}))
```

Comment:
This needs to be a string in data coming from INI, and it gets converted to boolean from the string